packages/vps/spec

```diff
@@ -0,0 +1,8 @@
+---
+name: vps
+
+dependencies:
+  - golang_1.7
+
+files:
+- "**/*"
```

Comment:
similar comment here. I think you can be more specific and only copy `golang_1.7` files